examples/pgbouncer/README.md

```diff
@@ -0,0 +1,260 @@
+    secretName: <YOUR-SERVER-CERT-SECRET>
+    items:
+    - key: tls.crt
+      path: cert.pem
+    - key: tls.key
+      path: key.pem
+```
+
+Next, we specify volume mounts in our PgBouncer container where the secrets will
+be stored:
+
+> [`pgbouncer_deployment.yaml`](pgbouncer_deployment.yaml#L31-L41))
+
+``` yaml
+- name: pgbouncer
+  image: <PG-BOUNCER-CONTAINER>
+  ports:
+  - containerPort: 8080
+  volumeMounts:
+  - name: cacert
+    mountPath: "/etc/ca"
+    readOnly: true
+  - name: servercert
+    mountPath: "/etc/server"
+    readOnly: true
+```
+
+Then we configure PgBouncer through environment variables. Note: we use 5431 for
+`DB_PORT` to leave 5432 available.
+
+> [`pgbouncer_deployment.yaml`](pgbouncer_deployment.yaml#L31-L41))
+
+``` yaml
+env:
+- name: DB_HOST
+  value: "127.0.0.1"
+- name: DB_USER
+  valueFrom:
+    secretKeyRef:
+      name: <YOUR-DB-SECRET>
+      key: username
+- name: DB_PASSWORD
+  valueFrom:
+    secretKeyRef:
+      name: <YOUR-DB-SECRET>
+      key: password
+- name: DB_NAME
+  valueFrom:
+    secretKeyRef:
+      name: <YOUR-DB-SECRET>
+      key: database
+- name: DB_PORT
+  value: "5431"
+- name: CLIENT_TLS_SSLMODE
+  value: "require"
+- name: CLIENT_TLS_CA_FILE
+  value: "/etc/ca/cert.pem"
+- name: CLIENT_TLS_KEY_FILE
+  value: "/etc/server/key.pem"
+- name: CLIENT_TLS_CERT_FILE
+  value: "/etc/server/cert.pem"
+```
+
+For the PgBouncer deployment, we add the proxy as a sidecar, starting it on port
+5431:
+
+> [`pgbouncer_deployment.yaml`](pgbouncer_deployment.yaml#L70-L76))
+
+``` yaml
+- name: cloud-sql-proxy
+  image: gcr.io/cloudsql-docker/gce-proxy:1.22.0
+  command:
+    - "/cloud_sql_proxy"
+    - "-instances=<INSTANCE_CONNECTION_NAME>=tcp:5431"
+  securityContext:
+    runAsNonRoot: true
+```
+
+Next, we create a PgBouncer service, listening on port 5342:
+
+> [`pgbouncer_service.yaml`](pgbouncer_service.yaml#L1-L11))
+
+``` yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: <YOUR-SERVICE-NAME>
+spec:
+  selector:
+    app: <YOUR-APPLICATION-NAME>
+  ports:
+  - protocol: TCP
+    port: 5432
+    targetPort: 5432
+```
+
+With the PgBouncer service and deployment done, we are ready to point our
+application at it.
+
+## Point your application at the service
+
+First, we configure a volume for the CA certificate, mapping the file name to
+`cert.pem`.
+
+> [`deployment.yaml`](deplyment.yaml#L1-L11))
+
+``` yaml
+volumes:
+- name: cacert
+  secret:
+    secretName: <YOUR-CA-CERT>
+    items:
+    - key: tls.crt
+      path: cert.pem
+```
+
+Next, we mount the volume within the application container:
+
+> [`deployment.yaml`](deplyment.yaml#L28-L31))
+
+``` yaml
+volumeMounts:
+- name: cacert
+  mountPath: "/etc/ca"
+  readOnly: true
+```
+
+Then, we configure environment variables for connecting to the database, this
+time including a `CA_CERT`:
+
+> [`deployment.yaml`](deplyment.yaml#L32-L53))
+
+``` yaml
+env:
+- name: DB_HOST
+  value: "<YOUR-SERVICE-NAME>.default.svc.cluster.local"
```

Comment:
This uses the default namespace, which might trip some folks up

Reply:
Added a comment to catch the eye.